src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1554,6 +1560,32 @@ internal static void ApplyConfiguration(JObject config, ScriptHostConfiguration
                 }
             }
 
+            JObject languageWorkerSection = (JObject)config["languageWorker"];
+            value = null;
+            if (languageWorkerSection != null)
+            {
+                if (languageWorkerSection.TryGetValue("maxMessageLength", out value))
+                {
+                    var requestedGrpcMaxMessageLength = int.Parse((string)value);
+                    if (ScriptSettingsManager.Instance.IsDynamicSku)
+                    {
+                        string message = $"Cannot set {nameof(scriptConfig.MaxMessageLength)} on Consumption plan. Default MaxMessageLength: {DefaultMaxMessageLengthDynamicSku} will be used";
+                        logger?.LogWarning(message);
+                    }
+                    else
+                    {
+                        scriptConfig.MaxMessageLength = requestedGrpcMaxMessageLength * 1024 * 1024;
```

Comment:
The multiplication here has me confused.. is the unit of the setting in the json file in MB? Lets make it explicit in the name as discussed above.